### PR TITLE
WINDUPRULE-194: adding jndi recognition from remote-binding

### PR DESCRIPTION
**Pull request by mareknovotny**
https://issues.jboss.org/browse/WINDUPRULE-194

### CODE REVIEW

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java [hunk 1]
```diff
@@ -136,10 +136,11 @@ public void perform(GraphRewrite event, EvaluationContext context, XmlFileModel
             if (StringUtils.isNotBlank(ejbName))
             {
                 LOG.info("Looking up name: " + ejbName);
-                for (EjbSessionBeanModel ejb : ejbSessionBeanService.findAllByProperty(EjbMessageDrivenModel.EJB_BEAN_NAME, ejbName))
+                for (EjbSessionBeanModel ejb : ejbSessionBeanService.findAllByProperty(EjbSessionBeanModel.EJB_BEAN_NAME, ejbName))
```
Comment (mareknovotny) [inline rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java]: This was a mistake I guess.

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java [hunk 2]
```diff
@@ -136,10 +136,11 @@ public void perform(GraphRewrite event, EvaluationContext context, XmlFileModel
...
                 {
                     String jndi = $(ejbRef).child("jndi-name").content();
                     String localJNDI = $(ejbRef).child("local-jndi-name").content();
+                    String remoteBindingJNDI = $(ejbRef).child("remote-binding").child("jndi-name").content();
```
Comment (mareknovotny) [inline rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java]: I was looking too long but this is just another type of jndi to use as global one.

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java (hunk 1 above)
Comment (jsight) [reply rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java]: Arguably, it should actually be org.jboss.windup.rules.apps.javaee.model.EjbBeanBaseModel#EJB_BEAN_NAME.

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java (hunk 2 above)
Comment (jsight) [reply rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/jboss/ResolveJBossLegacyEjbXmlRuleProvider.java]: Ah, good find.

**Pull request merged**